enhancements/installer/telemeter-metrics.md

```diff
@@ -0,0 +1,300 @@
+---
+title: installer-telemetry-metrics
+authors:
+  - "@patrickdillon"
+  - "@rna-afk"
+reviewers:
+  - "@abhinavdahiya"
+  - "@brancz"
+  - "@crawford"
+  - "@sdodson"
+approvers:
+  - "@abhinavdahiya"
+  - "@brancz"
+  - "@crawford"
+  - "@sdodson"
+creation-date: 2020-02-07
+last-updated: 2020-02-07
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# Installer Telemetry Metrics
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+ > 1. This requires creating an [aggregation pushgateway][weaveworks-aggregation-pushgateway] where the installer would push metrics. How would this be handled and by which team?
```

Comment:
presumably the app-sre team, you're going to want to talk to them quickly though because their epic planning either just happened or is about to happen

Reply:
this might make sense to live in the telemeter-server which authenticates all other telemetry push requests already today

Reply:
I think this needs a rollout plan. However I would suggest that:

1) we add the instrumentation and just play around with the instrumentation and a local push gateway to understand it's use and intricacies (no production systems needed until we figure this out)
2) once validated everything works as we imagine we deploy the push aggregation gateway as part of openshift telemetry, but only configure CI to actually push to it to get some experience operating it
3) once we are comfortable running this as a production system we default to sending metrics to our production instance on every install

Reply:
> presumably the app-sre team

I reached out to @pbergene 

> I think this needs a rollout plan. However I would suggest that:

That rollout plan sounds good. We can try to incorporate.